host/Cargo.toml

```diff
@@ -14,9 +14,13 @@ tracing = { workspace = true }
 wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", rev = "8d3a881b524d56498dfc71b65818b2abbeb1da44", features = ["component-model"] }
 wasi-common = { path = "../wasi-common" }
 wasi-cap-std-sync = { path = "../wasi-common/cap-std-sync" }
+reqwest = { version = "0.11.14", optional = true }
```

Comment:
It doesn't look like we're using any of this, except for the conversions to/from the reqwest types - so I think we should just drop the dependency. When it comes time to fill in more of the implementation, then we can add whatever dep is appropriate.